### ISSUE TITLE
Finally, I ve return to my old and stable Catalina. I hope new EFI will works fine. My issues was green screen on Apple TV and the usb wireless won't work, only ethernet . With Catalina works fine. Regards.

### ISSUE THREAD

**Issue by Gizmo648** [willza3/macOS-strix-B450i#27]
Finally, I ve return to my old and stable Catalina. I hope new EFI will works fine. My issues was green screen on Apple TV and the usb wireless won't work, only ethernet . With Catalina works fine. Regards.

_Originally posted by @Gizmo648 in https://github.com/willza3/macOS-strix-B450i/issues/25#issuecomment-751939256_

**Comment by willza3** [comment]
🥳  You'll be pleased to know this is finally fixed. Let me know if the latest EFI solves this issue for you.

**Comment by Gizmo648** [comment]
> 🥳 You'll be pleased to know this is finally fixed. Let me know if the latest EFI solves this issue for you.

Thanks a lot. Let me try again with this EFI
👍🏻☺️

**Comment by Gizmo648** [comment]
> 🥳 You'll be pleased to know this is finally fixed. Let me know if the latest EFI solves this issue for you.

Sorry , I updated this EFI but the screen stills green.
I hope there is a solution.

**Comment by willza3** [comment]
What GPU are you using?

**Comment by Gizmo648** [comment]
> 
> 
> What GPU are you using?

Its a Saphire RX570 Pulse 8GB.
Works great on Catalina.

**Comment by Gizmo648** [comment]
I found the fix for green screen .
Works ok on Big Sur.

This is the link

https://imgur.com/a/KbMlVZd

Thanks and regards.
